store/store.md

```diff
@@ -1 +1,60 @@
 # Store
+
+## Abstract
+
+The Store interface defines methods for storing and retrieving blocks, commits, and the state of the blockchain.
+
+## Protocol/Component Description
+
+The Store interface defines the following methods:
+
+- `Height`: Returns the height of the highest block in the store.
+- `SetHeight`: Sets the height saved in the Store if it is higher than the existing height.
```

Comment:
sets the height where?

Reply:
It takes in a height and updates it in the store only if the height is higher than what is saved currently. Updated the description to make it clearer.